### PR TITLE
NEXT-10072 - Plugin compatibility

### PR DESCRIPTION
**Pull request by swDennis**

### CODE REVIEW

File: tests/UsedClassesAvailableTest.php [hunk 1]
```diff
@@ -12,26 +12,16 @@ class UsedClassesAvailableTest extends TestCase
...
-            $classRelativePath = str_replace(['.php', '/'], ['', '\\'], $class->getRelativePathname());
-
-            $this->getMockBuilder($namespace . '\\' . $classRelativePath)
-                ->disableOriginalConstructor()
-                ->getMock();
-        }
-
-        // Nothing broke so far, classes seem to be instantiable
-        $this->assertTrue(true);
+        static::assertCount(1, $files);
```
Comment (mitelg) [inline tests/UsedClassesAvailableTest.php:17]: now you only check, if the classes are present. before there was a check if they were instantiatable

**Review by mitelg — CHANGES_REQUESTED** [review]

**Comment by mitelg** [comment]
maybe we could add a small e2e test to check if the tab is present in the admin

File: tests/UsedClassesAvailableTest.php (hunk 1 above)
Comment (swDennis) [reply tests/UsedClassesAvailableTest.php:17]: there is only 1 twig file... no php files

File: Resources/app/StoreFront/test/e2e/cypress/integration/extended-block.spec.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// / <reference types="Cypress" />
+describe('PluginExtendJsPlugin: Test is extended', () => {
+
+    it('checks for correct alert text', () => {
+        cy.visit('/');
+
+        cy.contains('Hello world!').should('be.visible');
```
Comment (mitelg) [inline Resources/app/StoreFront/test/e2e/cypress/integration/extended-block.spec.js:7]: shouldn't we check for the correct div element or something similar? or is this not important?

**Review by mitelg — CHANGES_REQUESTED** [review]
same comments as here: https://github.com/shopware/swag-docs-extend-js-plugin/pull/5

**Review by mitelg — APPROVED** [review]

**Pull request merged**